lib/Ama/Model/Votes.pm

```diff
@@ -3,12 +3,13 @@ use Mojo::Base -base;
 
 has 'pg';
 has 'username';
+has 'threshold';
```

Comment:
Can you be more descriptive / specific than threshold?  Threshold for what?  What if you want to add another threshold for something else in the future?  The trick, however, isn't to just extend the length of the name to 'downvote_threshold'.  The trick is to find a better word.  :D